content/api/resources/transactions/object.mdx

```diff
@@ -69,10 +69,40 @@ When Horizon returns information about a transaction, it uses the following form
   - An array of signatures used to sign this transaction.
 - valid_after
   - RFC3339 date-time string
-  - The date after which a transaction is valid.
+  - The date after which a transaction is valid. This will be deprecated in Horizon v3 in lieu of `preconditions.time_bounds.min_time`.
 - valid_before
   - RFC3339 date-time string
-  - The date before which a transaction is valid.
+  - The date before which a transaction is valid. This will be deprecated in Horizon v3 in lieu of `preconditions.time_bounds.max_time`.
+- preconditions
+  - object
+  - A set of transaction preconditions determining its validity.
+    - time_bounds
+      - min_time
+        - string, encoding a 64-bit UNIX timestamp
+        - The timestamp after which a transaction is valid.
+      - max_time
+        - string, encoding a 64-bit UNIX timestamp
+        - The timestamp before which a transaction is valid.
```

Comment:
Suggest including signedness that will affect how people parse the string.
```suggestion
      - min_time
        - string, containing an unsigned 64-bit integer UNIX timestamp
        - The timestamp after which a transaction is valid.
      - max_time
        - string, containing an unsigned 64-bit integer UNIX timestamp
        - The timestamp before which a transaction is valid.
```

Reply:
Yeah, good call. Putting these into the description, though, as putting them into the type makes the rendering ugly and doesn't align with how it's done elsewhere.

Reply:
OpenAPI spec when? :cricket: